source/OptimaJet.DWKit.StarterApplication/Services/HttpCurrentUserContext.cs

```diff
@@ -26,5 +68,69 @@ public class HttpCurrentUserContext : ICurrentUserContext
             }
         }
 
+        public string Email {
+            get {
+                if (email == null) {
+                    email = this.HttpContext.GetAuth0Email();
+                }
+                return email;
+            }
+        }
+
+        public string GivenName {
+            get {
+                if (givenName == null) {
+                    var auth = AuthType;
+                    if (string.Compare(auth, "auth0", StringComparison.Ordinal) == 0) {
```

Comment:
there is `StartsWith`? https://msdn.microsoft.com/en-us/library/baketfxw(v=vs.110).aspx

Reply:
I started with `CompareTo` but VS suggested to change it to this.

Reply:
I mean, I think you can just do `auth.StartsWith("auth0")`